### PR TITLE
E225 Check for space around boolean operators

### PR DESCRIPTION
**Pull request by adamchainz**
This was documented in the docstring for `missing_whitespace_around_operator` but not implemented, allowing e.g. `1and 0` to pass. Fixed by adding test case then modifying the check to search for the 'and' and 'or' operators as well.

### CODE REVIEW

File: testsuite/E22.py [hunk 1]
```diff
@@ -76,6 +76,10 @@
 i=i+ 1
 #: E225 E225
 i=i +1
+#: E225
+i = 1and 1
+#: E225
+i = 1or 0
```
Comment (asottile) [inline testsuite/E22.py]: here's a few more:

```python
1is 1
1not in []
1in []
```

Comment (adamchainz) [reply testsuite/E22.py]: Added support for 'is' and 'not', updating for the two-word 'not in' is beyond me right now (07:30 after a long haul flight...)

Comment (asottile) [reply testsuite/E22.py]: missed `not` ;)

**Comment by adamchainz** [comment]
Thanks @asottile for review and @sigmavirus24 for merge!

**Comment by sigmavirus24** [comment]
Thanks for the fix! When you're less jet-lagged and feel like puzzling out `not in`, feel free to send another PR.

**Pull request merged**